common/api.ts

```diff
@@ -0,0 +1,30 @@
+import { updateToken } from "../stores/slices/userSlice";
+import localStorage from "./localStorage";
+
```

Comment:
Is this file finally been used? Or can we delete it?

Reply:
yess, we are using the localStorage of the phone to store/get the token